Chinese/Demo_htx/demo.md

```diff
@@ -0,0 +1,3 @@
+click [here](http://www.baidu.com) to visit baidu
+
+click [here](language.md) to visit raw url
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.